zboxcore/sdk/networkworker_test.go

```diff
@@ -0,0 +1,216 @@
+package sdk
```

Comment:
Can you test these functions too?
CreateAllocationForOwner()
GetAllocationMinLock()